drift/openapi/api.spec.yaml

```diff
@@ -11,21 +11,49 @@ info:
 servers:
   - url: /r/insights/platform/drift/v0
 
+
 paths:
   /status:
     get:
-      summary: health check call
+      summary: health check liveness call
       operationId: drift.views.v0.status
       tags:
         - status
       responses:
         '200':
-          description: a small JSON indicating the application is deployed
+          description: "A small JSON indicating the application is deployed.
+                        This serves as both the liveness and readiness call."
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/Status"
 
+  /comparison_report:
+    parameters:
+      - $ref: '#/components/parameters/rhIdentityHeader'
+    get:
+      summary: fetch comparison report
+      description: "Fetch a comparison report for given system IDs. Currently,
```

Comment:
I think we could drop `currently` here. Let's not commit to 'future' language if we can avoid it.